Writerside/topics/channel-kick.md

```diff
@@ -0,0 +1,13 @@
+# Mitglieder aus einem Textkanal entfernen
+
+## So kannst du Spieler aus deinem Textkanal entfernen:
+
+<deflist>
+<def title="Mitglieder entfernen">
+Um Mitglieder aus deinem Textkanal zu entfernen, benutze den Befehl <code>/channel kick <Spieler></code>.
+Füge anschließend den Spieler hinten an, welchen du entfernen möchtest.
+<tip>
+Zum Beispiel: <code>/channel kick Twisti_Twixi</code>
+</tip>
```

Comment:
Hier vllt noch ein Tipp oder Info Box oder so (ich bin gerade am Handy da kann ich das nicht machen) das man der Owner sein muss um die Aktion auszuführen

Reply:
Was meinst du mit Tipp oder Info Box? da ist doch ein Beispiel als Tipp in einer Box

Reply:
Ich meinte das gesagt wird das man der owner von diesem Channel sein muss um die Aktion auszuführen und das halt vllt in so einer Box. Kommt drauf an wie das dann aussieht